tool/check_gemini_policy.sh

```
@@ -0,0 +1,63 @@
+    # If the previous block matched both conditions, we are done searching
+    if [[ "$is_make" == true && "$is_allow" == true ]]; then
+      break
+    fi
+    # Reset for the new block
+    is_make=false
+    is_allow=false
+    commands_str=""
+  fi
+  
+  # Check for commandRegex matching 'make' and extract the group in parentheses
+  if [[ "$line" =~ commandRegex.*make.*\(([^)]+)\) ]]; then
```

Shellcheck (0.11.0)

[info] The mentioned syntax error was in this if expression.

(SC1009)